packages/block-editor/src/components/rich-text/README.md

```diff
@@ -37,9 +37,9 @@ Render a rich [`contenteditable` input](https://developer.mozilla.org/en-US/docs
 
 *Optional.* Called when the block can be removed. `forward` is true when the selection is expected to move to the next block, false to the previous block.
 
-### `formattingControls: Array`
+### `disallowFormats: Array | Boolean`
```

Comment:
What's the reasoning for the API change?

Reply:
I hoped we can fix it by filtering out all formatting controls which aren't listed in this array.

Reply:
Doing the opposite is harder. E.g. for the button block, you only really want to omit the link button, not all the ones in the dropdown. Either you choose to omit some specified formats (or maybe HTML tags), or you omit all of them (no formatting allowed). Block authors are not always aware of buttons that other plugins might add, so if I have a plugin that enables sub and superscript, the block author is preventing me from adding any formatting options.

Reply:
Other idea: have two different props for different purposes: `disallowFormats` to disable all formats and `disableTabbableFormats` to disable tabbable formats only, to be used e.g. inside a link, label, button...

Reply:
Ideally, it would be cool to auto-detect from the parent elements which formats would be valid children and only enable those.

Reply:
Got it, just be mindful of the breaking changes and if it's inevitable, we should warn and document properly (probably a devnote etc...)

Reply:
Solved. Deprecated `formattingControls`.